config/locales/components/candidate_interface/editable_section_warning.yml

```diff
@@ -0,0 +1,5 @@
+en:
+  editable_section_warning:
+    personal_statement: Any changes you make to your personal statement will not be included in applications you have already submitted.
+    default: Any changes you make will be included in applications you have already submitted.
+    work_history: Any changes you make will not be included in applications you have already submitted.
```

Comment:
Want to double check that Content Designers have seen this?

Reply:
Cheked it with Rose, she approves 👍🏻